Add "Import Video" to the sign language tool (BL-5871)

StephenMcConnel opened the pull request:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/bloombooks/bloomdesktop/2499)
<!-- Reviewable:end -->

andrew-polk commented:
Reviewed 5 of 5 files at r1.
Review status: all files reviewed, 2 unresolved discussions (waiting on @StephenMcConnel)

---

*[src/BloomExe/web/controllers/SignLanguageApi.cs, line 104 at r1](https://reviewable.io/reviews/bloombooks/bloomdesktop/2499#-LF-H-fw-zj7f6tsUtUn:-LF-H-fw-zj7f6tsUtUo:b-qth9ts) ([raw file](https://github.com/bloombooks/bloomdesktop/blob/bcd848004d4c3040137fb43b3367f25814d3cb8e/src/BloomExe/web/controllers/SignLanguageApi.cs#L104)):*
> ```Smalltalk
> 				// Enhance: if we end up needing this it should be localizable. But the current plan is to disable
> 				// video recording if there is no container on the page.
> 				var msg = "There's nowhere to put a video on this page." +
> ```

Based on the comment above, we either should localize this, or the button for import should be disabled.
Or am I missing something?

---

*[src/BloomExe/web/controllers/SignLanguageApi.cs, line 272 at r1](https://reviewable.io/reviews/bloombooks/bloomdesktop/2499#-LF-GF-b7bW2qD7nU0SC:-LF-GF-b7bW2qD7nU0SD:b-c5neer) ([raw file](https://github.com/bloombooks/bloomdesktop/blob/bcd848004d4c3040137fb43b3367f25814d3cb8e/src/BloomExe/web/controllers/SignLanguageApi.cs#L272)):*
> ```
> video video
> ```
typo typo

---


*Comments from [Reviewable](https://reviewable.io/reviews/bloombooks/bloomdesktop/2499)*
<!-- Sent from Reviewable.io -->

StephenMcConnel commented:
Review status: 3 of 5 files reviewed, 2 unresolved discussions (waiting on @andrew-polk and @StephenMcConnel)

---

*[src/BloomExe/web/controllers/SignLanguageApi.cs, line 104 at r1](https://reviewable.io/reviews/bloombooks/bloomdesktop/2499#-LF-H-fw-zj7f6tsUtUn:-LF-NNDE6nQU_l_H7dv1:b9fhdti) ([raw file](https://github.com/bloombooks/bloomdesktop/blob/bcd848004d4c3040137fb43b3367f25814d3cb8e/src/BloomExe/web/controllers/SignLanguageApi.cs#L104)):*
<details><summary><i>Previously, andrew-polk wrote…</i></summary><blockquote>

Based on the comment above, we either should localize this, or the button for import should be disabled.
Or am I missing something?
</blockquote></details>

The button for import should be disabled if there is no container on the page.  That's how I interpret the comment, especially after editing to clarify.  I've fixed the code so that import is actually disabled in those cases.

---

*[src/BloomExe/web/controllers/SignLanguageApi.cs, line 272 at r1](https://reviewable.io/reviews/bloombooks/bloomdesktop/2499#-LF-GF-b7bW2qD7nU0SC:-LF-NKxw5t4AMvBg7a8U:b-auecvn) ([raw file](https://github.com/bloombooks/bloomdesktop/blob/bcd848004d4c3040137fb43b3367f25814d3cb8e/src/BloomExe/web/controllers/SignLanguageApi.cs#L272)):*
<details><summary><i>Previously, andrew-polk wrote…</i></summary><blockquote>

> ```
> video video
> ```
typo typo
</blockquote></details>

oops oops.  Fixed.

---


*Comments from [Reviewable](https://reviewable.io/reviews/bloombooks/bloomdesktop/2499)*
<!-- Sent from Reviewable.io -->

andrew-polk commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Reviewed 2 of 2 files at r2.
Review status: :shipit: complete! all files reviewed, all discussions resolved

---



*Comments from [Reviewable](https://reviewable.io/reviews/bloombooks/bloomdesktop/2499#-:-LF-WTYjCcqwREorYOo8:bnfp4nl)*
<!-- Sent from Reviewable.io -->
